packages/shorebird_cli/lib/src/commands/release_command.dart

```diff
@@ -248,6 +248,19 @@ ${summary.join('\n')}
       }
     }
 
+    try {
+      await codePushClient.createReleaseArtifact(
+        releaseId: release.id,
+        artifactPath: bundlePath,
+        arch: 'universal',
```

Comment:
I might call it "multi" or "aab", we don't know what archs are in it.  We could tag it with the archs if we wanted. 🤷   e.g. 'arm-arm64-x86'

Reply:
Renamed to "aab"